int32 radarNumber
---
float32 distanceToHome
float32 deltaDistanceToHome
float32 velocityFromHome
